src/autolabel/tasks/attribute_extraction.py

```diff
@@ -235,7 +235,16 @@ def parse_llm_response(
                     error_message=str(e),
                 )
 
-        # TODO(rajas): Handle output guidelines not followed error (for options case)
+        if successfully_labeled:
+            for attribute in self.config.attributes():
+                attr_options = attribute.get("options")
+                if attr_options is not None and len(attr_options) > 0:
+                    attr_label = llm_label.get(attribute["name"])
+                    if attr_label is not None and attr_label not in attr_options:
+                        logger.warning(
```

Comment:
not returning an error?

Reply:
yeah since errors are at an LLMAnnotation level, it would require either returning no label, or filtering out the invalid attributes (as is done here).